src/base/StringLiteral.zig

```diff
@@ -33,6 +33,13 @@ pub const Store = struct {
     /// continues to the previous byte
     buffer: std.ArrayListUnmanaged(u8) = .{},
 
+    /// Intiizalizes a `StringLiteral.Store` with capacity `bytes` of space.
+    pub fn initCapacityBytes(gpa: std.mem.Allocator, bytes: usize) Store {
```

Comment:
Note: I named this special cause it is a less intuitive size. It isn't capacity for 256 strings. It is capacity for 256 bytes and however many strings that ends up being.

Reply:
It wouldn't hurt to add this "why" explanation to the doc comment then. Once you explain the bytes vs string thing that makes sense to me.